### ISSUE TITLE
Fork and Github Actions

### ISSUE THREAD

**Issue by sandrojr1991** [DADi590/Console-news-on-consoles#2]
Hello,

I saw this on HAN Toolbox and it is really nice to see the psx-place news direcly on the console.

I forked your repository to learn how it is made and make some modifications on it.

I'm new to github and stuff, can you teach me how to reproduce your github actions? i didn't understande a thing :P

**Comment by DADi590** [comment]
Hi! Forget about the Actions, I didn't even know there ones here. I haven't seen about what they are yet --> so I have also no idea how they're here. I must have enabled something accidentally. Forget about them, I don't know what they do xD (and I just deleted everything on that page now).

**Comment by sandrojr1991** [comment]
I thought that the atomization is being made by it, so how do updates get done? manually?

**Comment by DADi590** [comment]
No, nothing is automated here. I didn't even setup a Git repo on my computer for it. I had never touched on Git when I released my first version of this toolbox. I update it manually by having a copy of the last release, I edit that copy, and I upload all the files in a ZIP when that's ready.

It's in a ZIP because as I have nothing automated, if I want to delete stuff, it's infinity to delete multiple files manually, so I just put all in a ZIP. Bad Git practices xD.

**Comment by sandrojr1991** [comment]
Got it, but isnt there a way to make it work with git action? recently @bucanero made the github actions compile a new pkg once a new commit is made, automatically.

**Comment by DADi590** [comment]
It's possible, but I don't know how Actions work here (or what they can do, actually), nor I do really have time to check it. Sorry about that. In my case it wasn't really needed because I only release one of these once in some time (infinity), so I never bothered to automate anything here.
